AUTH.md

```diff
@@ -167,9 +167,7 @@ To use Azure keyvault, create your keyvault (above named `witsmlexp-servers-kv`)
  
 ## Serverlist
 
-Credentials will be mapped on URL from secrets with the serverlist. `Server` entry in MongoDB or CosmosDB will have property `securityscheme` that can be `Basic` or `OAuth2`
-
-The app role assigned to a server will be compared to the role claims in the JWT provided in the Authorization header. If a user has been assigned the same application role, system credentials will be made available to the user. An API call will use the system credentials if the system username is set in the `WitsmlTargetUsername` or `WitsmlSourceUsername` header.
+Credentials will be mapped on URL from secrets with the serverlist. `Server` entry in MongoDB or CosmosDB will have the property `roles`. The app role assigned to a server will be compared to the role claims in the JWT provided in the Authorization header. If a user has been assigned the same application role, system credentials will be made available to the user. An API call will use the system credentials if the system username is set in the `WitsmlTargetUsername` or `WitsmlSourceUsername` header.
```

Comment:
We might add that if frontend and backend has been started with `OAuth2Enabled=false` in the backend and empty `MSALENABLED=` the roles will be ignored, what do you think?